### PR TITLE
#58 - Design PR template proposal

### LINKED ISSUE #58: Design PR template proposal

### PR DESCRIPTION
**Pull request by miroslavpojer**
Release Notes:
- Implemented a new template for Pull requests.

Closes #58

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+## Checklist:
+> Ensure the following tasks are completed before submission.
+- [ ] My code follows the project style guidelines
+- [ ] I have performed a self-review of my code
+- [ ] I have commented my code, particularly in hard-to-understand areas
+- [ ] I have added tests that prove my fix is effective or my feature works
+- [ ] New and existing tests pass locally with my changes
+- [ ] I have used a code formatting tool to ensure consistent style
+- [ ] I have run a linter to check for code quality issues
```
Comment (MobiTikula) [inline .github/pull_request_template.md:12]: Great idea of having this template. Just add a period at the end of each sentence.

**Review by MobiTikula — COMMENTED** [review]
Please implement a tiny comment.

File: .github/pull_request_template.md (hunk 1 above)
Comment (miroslavpojer) [reply .github/pull_request_template.md]: Fixed in commit 03dcec8f097eeadc82b4726df46404de52d3be47.

**Review by MobiTikula — APPROVED** [review]
Approved.

**Pull request merged**